src/lib/utils/index.ts

```diff
@@ -15,8 +15,8 @@ export const maybeCDN = (uri: string) => {
 export const getEnvironmentVariable = (environmentVariable: string): string => {
   const unvalidatedEnvironmentVariable = process.env[environmentVariable];
   if (!unvalidatedEnvironmentVariable) {
-    return '';
     console.error(`Couldn't find env var: ${environmentVariable}`);
+    return '';
```

Comment:
IME environment variables should be considered prerequisites for startup/functioning, so a missing environment variable should lead to complete failure, or at least throw an error that must be handled by the caller.

Reply:
This is some crossmint stuff Milo made, he can fix it later :)

Reply:
my fault here lmao, sorry. it was more like a debugging thing :D